libraries/BluetoothSerial/src/BTAddress.h

```diff
@@ -24,12 +24,12 @@ class BTAddress {
 public:
```

Comment:
up in the includes, you need to change `#include <string>` to `#include "Arduino.h"`